plasmapy/formulary/parameters.py

```diff
@@ -507,15 +536,43 @@ def thermal_speed(
     """
     m = mass if np.isfinite(mass) else particles.particle_mass(particle)
 
+    # TODO: Could implement this more cleanly for any integer ndim
+    # at the cost of using the gamma function in the expression for the
+    # mean magnitude velocity.
+
```

Comment:
Sure, but it'd be slower, too.

Reply:
Fair enough - 3D plasmas are more than complicated enough for me.